README.md

```diff
@@ -1,3 +1,14 @@
 # LocalGov Drupal core functionality
 
 LocalGov Drupal Core module, for helper functions and core dependencies.
+
+## Default blocks
+This module contains a mechanism that will place default blocks into your site's
+active theme when other localgov modules are installed. This is indended to 
```

Comment:
Typo /indended/intended/.

Reply:
No, you're not missing anything. I hadn't thought to test this in the installer, to be honest. Good find! I can recreate this locally, so I'll sort it out now.

Reply:
So it turns out 'core' is a kind of fake theme that Drupal will return if there's no other themes: https://git.drupalcode.org/project/drupal/-/blob/11.x/core/lib/Drupal/Core/Theme/ThemeInitialization.php?ref_type=heads#L99

Reply:
So to fix this, I've added a couple of special cases that check if we're in the installer. One to stop trying to load the active theme, as that's not a thing in the installer, and skip the check that the themes exist, because none of them do yet.

The other skips the check that the theme has the requested regions, as we can't load the theme info to check that either.

I think doing it this way is fine for the installer case, as in practice it'll be mostly be people who are using the default project, where we know what exists and what doesn't.

Reply:
If you know that get active theme is going to return `core` in all cases where there isn't an active real theme, is it not better to check the active theme, and if it is `core` bail out? Just in case it happens on some other occasion too?

Reply:
Which reminds me.. The other thing I did was move the check for themes existing below the code that finds the current default. (It was the other way round before). That should handle that case. Although I suspect the only time you'll ever have no theme while installing a module is going to be the installer.

Reply:
I'm going to guess it's only the installer... but... maybe some new sync' recipe or something that doesn't use the installer but something else... I don't know. It feels like it is an unnecessary assumption if it can bail out on the fact that the theme isn't there no matter what?

Reply:
Well, if such an installer-like thing does exist in the future, the code as-is will add 'core' (or whatever) to the list of themes, then the next block of code checks all the themes exist, at which point, it'll be taken out of the list. So I think we're good for the moment.

Reply:
Ok, I've changed my mind. Latest commit wraps the call to $this->themeManager->getActiveTheme()->getName(); in a new method called getActiveThemeName() and returns null if the theme name is 'core' to indicate that there's no real active theme.

Reply:
:-)